<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceptionDashboard</class>
 <widget class="QMainWindow" name="ReceptionDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Espace R√©ception</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- STYLE GLOBAL --- */
QMainWindow { background-color: #f0f2f5; }
QWidget { font-family: &quot;Segoe UI&quot;, sans-serif; }

/* --- SIDEBAR --- */
QFrame#frame_sidebar { background-color: white; border-right: 1px solid #e0e0e0; }
QLabel#label_app_name { color: #2d7aed; font-weight: bold; font-size: 18px; padding: 10px; }
QLabel#label_role { color: #777; font-size: 13px; padding-bottom: 20px; }

/* BOUTONS SIDEBAR */
QPushButton.SidebarBtn {
    text-align: left; padding: 12px 20px; border: none; background-color: transparent;
    font-size: 15px; color: #555; border-radius: 5px; margin: 4px 10px;
}
QPushButton.SidebarBtn:hover { background-color: #f5f7fa; color: #2d7aed; }
QPushButton.SidebarBtn:checked { background-color: #e6f0ff; color: #2d7aed; font-weight: bold; }

/* --- CONTENEURS --- */
QFrame.Card { background-color: white; border-radius: 8px; border: 1px solid #dcdcdc; }

/* --- TABLEAU (CORRECTION COULEUR) --- */
QTableWidget { 
    background-color: white; 
    border: none; 
    gridline-color: #f0f0f0; 
    selection-background-color: #e6f0ff; 
    selection-color: #2d7aed;
    color: #333333; /* &lt;-- ICI : Texte en noir */
}

QHeaderView::section { 
    background-color: #f8f9fa; 
    padding: 8px; 
    border: none; 
    border-bottom: 2px solid #2d7aed; 
    font-weight: bold; 
    color: #333333; /* &lt;-- ICI : Titres en noir */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_main">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    
    <item>
     <widget class="QFrame" name="frame_sidebar">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_sidebar">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="label_app_name">
         <property name="text">
          <string>üè• Accueil H√¥pital</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_role">
         <property name="text">
          <string>Service R√©ception</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_top">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_dashboard">
         <property name="class" stdset="0">
          <string>SidebarBtn</string>
         </property>
         <property name="text">
          <string>üìÖ  Agenda Global</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_patients">
         <property name="class" stdset="0">
          <string>SidebarBtn</string>
         </property>
         <property name="text">
          <string>üë§  Gestion Patients</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_rdv">
         <property name="class" stdset="0">
          <string>SidebarBtn</string>
         </property>
         <property name="text">
          <string>üìÜ  Prise de RDV</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nav_admissions">
         <property name="class" stdset="0">
          <string>SidebarBtn</string>
         </property>
         <property name="text">
          <string>üõèÔ∏è  Hospitalisation</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_bottom">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_logout">
         <property name="class" stdset="0">
          <string>SidebarBtn</string>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #d9534f;</string>
         </property>
         <property name="text">
          <string>üö™ D√©connexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      
      <widget class="QWidget" name="page_agenda">
       <layout class="QVBoxLayout" name="verticalLayout_agenda">
        <property name="leftMargin">
            <number>30</number>
        </property>
        <property name="topMargin">
            <number>30</number>
        </property>
        <property name="rightMargin">
            <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_title_agenda">
          <property name="styleSheet">
           <string notr="true">font-size: 22px; font-weight: bold; color: #333; margin-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>Rendez-vous de la journ√©e</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_agenda_list">
          <property name="class" stdset="0">
           <string>Card</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_table">
           <item>
            <widget class="QTableWidget" name="table_global_agenda">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Heure</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Patient</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>M√©decin</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Service</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Statut</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>